modern/src/common/util/formatter.js

```diff
@@ -64,6 +69,16 @@ export const formatNumericHours = (value, t) => {
   return `${hours} ${t('sharedHourAbbreviation')} ${minutes} ${t('sharedMinuteAbbreviation')}`;
 };
 
+export const formatDriverUniqueId = (value) => {
```

Comment:
I don't know about this one. We only do light formatting in these functions. A lookup like this should probably be in a different place.

Reply:
Was thinking the same initially, but then saw that it's implemented like that in the old interface and just decided to duplicate it. But I'm open to discuss how we can better represent driver(s)/mapping and just remove it from this PR/commits.

Reply:
Will tackle this at some later time then, removed it for now. Thanks for the guidance.